src/app/components/sidebar/sidebar.component.ts

```diff
@@ -16,6 +16,7 @@ export const ADMIN_ROUTE_ITEMS: RouteInfo[] = [
 export const REGULAR_USER_ROUTE_ITEMS: RouteInfo[] = [
     { path: '/dashboard', title: '首页', icon: 'dashboard', class: '' },
     { path: '/training-record/entry', title: '培训记录填报', icon: 'create', class: '' },
+    { path: '/training-event/table-list', title: '校内培训活动查看', icon: 'create', class: '' },
```

Comment:
In order to be consistent, I personally prefer '/training-event/events', this way the URL will be similar to backend APIs.